packages/server/graphql/mutations/updateJiraDimensionField.ts

```diff
@@ -100,21 +100,20 @@ const updateJiraDimensionField = {
     }
 
     const MAX_JIRA_DIMENSION_FIELDS = 100 // prevent a-holes from unbounded growth of the Team object
+    const sortedJiraDimensionFields = jiraDimensionFields
+      .slice(jiraDimensionFields.length - MAX_JIRA_DIMENSION_FIELDS)
+      .sort((a, b) => (JSON.stringify(a) < JSON.stringify(b) ? -1 : 1))
```

Comment:
`JSON.stringify` is not a stable stringifier, meaning the order of the objects is non-deterministic.
while it will _probably_ work for this node version & future versions of V8, it's not guaranteed & if the order ever does change, that'll be a tricky bug to find.
instead, use fast-json-stable-stringify